src/gen100/libgrb_api.jl

```diff
@@ -53,7 +64,7 @@ function GRBgetcharattrelement(model, attrname, element, valueP)
 end
 
 function GRBsetcharattrelement(model, attrname, element, newvalue)
-    ccall((:GRBsetcharattrelement, libgurobi), Cint, (Ptr{GRBmodel}, Ptr{Cchar}, Cint, UInt8), model, attrname, element, newvalue)
+    ccall((:GRBsetcharattrelement, libgurobi), Cint, (Ptr{GRBmodel}, Ptr{Cchar}, Cint, Cchar), model, attrname, element, newvalue)
```

Comment:
I think the test failure is because `Cchar` is not a `UInt8` on Mac, but an `Int8`.
```julia
julia> Cchar
Int8
```
Let me take a look.

Reply:
GH actions is showing the same failures though (on linux)?